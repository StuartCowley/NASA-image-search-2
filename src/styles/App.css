.App {
  display: flex;
  flex-direction: column;
  align-items: center;
}

.nasa-logo {
  height: auto;
  width: 50%;
}

@media screen and (min-width: 768px) {
}

@media screen and (min-width: 1024px) {
  .nasa-logo {
    width: 20%;
  }
}
