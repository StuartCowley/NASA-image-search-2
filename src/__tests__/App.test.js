import { render, screen } from "@testing-library/react";
import App from "../components/App";

describe("App", () => {
  test("renders component correctly", () => {
    const { asFragment } = render(<App />);

    expect(asFragment()).toMatchSnapshot();
  });
  test("renders logo", () => {
    render(<App />);
    const imgElement = screen.getByAltText("nasaLogo");

    expect(imgElement).toBeInTheDocument();
    expect(imgElement).toHaveClass("nasa-logo");
  });
});
