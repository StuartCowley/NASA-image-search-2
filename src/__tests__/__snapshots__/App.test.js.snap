// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders component correctly 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <img
      alt="nasaLogo"
      class="nasa-logo"
      src="https://cdn.cnn.com/cnnnext/dam/assets/200424060716-nasa-worm-logo.jpg"
    />
    <form
      class="search__form"
    >
      <input
        class="search__input"
        placeholder="moon"
        type="text"
      />
      <button
        class="search__button"
        type="submit"
      >
        Go!
      </button>
    </form>
    <p>
      No results
    </p>
  </div>
</DocumentFragment>
`;
